extension/src/utils/strings.ts

```diff
@@ -0,0 +1,18 @@
+const emojiMap: { [key: string]: string } = {
+  ':ice:': '🧊',
+  ':rocket:': '🚀',
+  ':bug:': '🐛',
+  ':microscope:': '🔬',
+  ':linked_paperclips:': '🔗',
+  ':chart_increasing:': '📈',
+  ':chart_decreasing:': '📉',
+  ':locked_with_key:': '🔒',
+  ':play_button:': '▶️',
+  ':check_mark:': '✅',
+  ':cross_mark:': '❌',
+  ':hammer_and_wrench:': '🛠️'
+};
+
+export function formatText(str: string): string {
+  return str.replace(/:\w+:/g, match => emojiMap[match] || match);
```

Comment:
I think you might be able to tighten this up a bit. Maybe this (but add test cases):

`:[a-z]+(?:_[a-z]+)*:`